### PR TITLE
Fix Linux and Windows templates for labels

### PR DESCRIPTION
**Pull request by mareklibra**
Labels for OfflineVirtualMachines fixed accroding to docummentation [1]
and sample OVM [2].

[1] https://raw.githubusercontent.com/kubevirt/kubevirt/master/docs/devel/offline-virtual-machine.md
[2] https://raw.githubusercontent.com/kubevirt/kubevirt/master/cluster/my-ovm.yaml

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: cluster/vm-template-rhel7.yaml [hunk 1]
```diff
@@ -53,4 +61,4 @@ parameters:
   value: 4096Mi
 - name: CPU_CORES
   description: Amount of cores
-  value: 4
+  value: "4"
```
Comment (rmohr) [inline cluster/vm-template-rhel7.yaml]: So it needs to be a string right?

Comment (mareklibra) [reply cluster/vm-template-rhel7.yaml]: Surprisingly yes for here, the `oc process` complains about non `"` character here.
Anyway, if this default is used with the former changes above, the processed value is without quotation marks (so integer)

Comment (mareklibra) [reply cluster/vm-template-rhel7.yaml]: Good to note: OpenShift v3.9.0-alpha.3

**Pull request merged**